src/main/webapp/app/shared/chart/active-students-chart.ts

```diff
@@ -59,8 +59,8 @@ export abstract class ActiveStudentsChart {
      * @protected
      */
     protected determineDifferenceBetweenIsoWeeks(date1: dayjs.Dayjs, date2: dayjs.Dayjs): number {
-        const normalizedDate1 = date1.day(0).hour(0).minute(0).second(0).millisecond(0);
-        const normalizedDate2 = date2.day(0).hour(0).minute(0).second(0).millisecond(0);
+        const normalizedDate1 = date1.isoWeekday(1).hour(2).minute(0).second(0).millisecond(0);
```

Comment:
Is the `hour(2)` a hack to make it work for CEST instead of UTC?

What happens during the winter CET times? What if a user is in a different time zone?

Would the use of something like `const offset = (24 + Math.ceil(dayjs().utcOffset() / 60)) % 24;` (https://day.js.org/docs/en/manipulate/utc-offset) instead make it more robust or that not needed?

Reply:
This evolved during my last investigation. In general, It is not important that the dates are set to 00:00 UTC. It is sufficient that both dates are set to the same hour. Therefore, this should be robust in all time zones.

Reply:
Sounds reasonable, thanks.